contracts/RewardManagerFactory.sol

```diff
@@ -0,0 +1,193 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.7.6;
+
+import "./RewardManager.sol";
+
+contract RewardManagerFactory is Ownable{
+    using SafeMath for uint256;
+    using SafeMath for uint128;
+
+    /// @notice all the information for this RewardManager in one struct
+    struct RewardManagerInfo {
+        address managerAddress;
+        uint256 startDistribution;
+        uint256 endDistribution;
+    }
+    
+    struct tempInfo{
+        uint256 _totalVested;
+        uint256 _totalDrawnAmount;
+        uint256 _amountBurnt;
+        uint256 _claimable;
+        uint256 _bonusRewards;
+        uint256 _stillDue;
+    }
+
+    RewardManagerInfo[] public managers;
+
+    uint256 public managerIndex;
```

Comment:
Why is this required exactly?

Reply:
this was being removed in previous commit